<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StokKartiForm</class>
 <widget class="QDialog" name="StokKartiForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Stok Kartı Bilgileri</string>
  </property>
  <widget class="QLabel" name="UrunResimlabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>827</x>
     <y>10</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/ui/box.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="KaydetpushButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/check.png</normaloff>:/images/ui/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="iptalpushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>İPTAL</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/remove.png</normaloff>:/images/ui/remove.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="temizlepushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TEMİZLE</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/eraser.png</normaloff>:/images/ui/eraser.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ResimEklepushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resim ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResimSilpushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resmi sil</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Detaylar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Stok Kodu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Stok Grubu</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>KDV</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Alış Fiyatı</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Stok Miktarı</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Barkod</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="BirimicomboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ad</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Tedarikçi</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Alt gruplar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="BarkodOlusturpushButton">
      <property name="text">
       <string>Barkod Oluştur</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="OTVspinBox"/>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="stokGruplaripushButton">
      <property name="text">
       <string>Stok grupları</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Üretici</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="AdlineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Birimi</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="AFiyatdoubleSpinBox">
      <property name="maximum">
       <double>9999999.990000000223517</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="BarkodlineEdit"/>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="SFiyatdoubleSpinBox">
      <property name="maximum">
       <double>9999999.990000000223517</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="MiktarlineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="StokKodlineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="StokGrubucomboBox"/>
    </item>
    <item row="14" column="1" colspan="2">
     <widget class="QComboBox" name="UreticicomboBox"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>OTV Dahil mi?</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="altGrupcomboBox"/>
    </item>
    <item row="12" column="1">
     <widget class="QComboBox" name="OTVDahilcomboBox">
      <item>
       <property name="text">
        <string>HAYIR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EVET</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>KDV Dahil mi?</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="KDVspinBox"/>
    </item>
    <item row="13" column="1" colspan="2">
     <widget class="QComboBox" name="TedarikcicomboBox"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Alt grup</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Satış Fiyatı</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="KDVDahilcomboBox">
      <item>
       <property name="text">
        <string>HAYIR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EVET</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>OTV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cikispushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ÇIKIŞ</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/cikis.png</normaloff>:/images/ui/cikis.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="stokHareketleripushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stok hareketleri</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/chart-100.png</normaloff>:/images/ui/chart-100.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="stokGirpushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stok gir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stokDuspushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stok düş</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>KaydetpushButton</zorder>
  <zorder>iptalpushButton</zorder>
  <zorder>temizlepushButton</zorder>
  <zorder>ResimEklepushButton</zorder>
  <zorder>ResimSilpushButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>UrunResimlabel</zorder>
  <zorder>cikispushButton</zorder>
  <zorder>stokHareketleripushButton</zorder>
  <zorder>stokGirpushButton</zorder>
  <zorder>stokDuspushButton</zorder>
 </widget>
 <tabstops>
  <tabstop>BarkodlineEdit</tabstop>
  <tabstop>StokKodlineEdit</tabstop>
  <tabstop>AdlineEdit</tabstop>
  <tabstop>BirimicomboBox</tabstop>
  <tabstop>MiktarlineEdit</tabstop>
  <tabstop>StokGrubucomboBox</tabstop>
  <tabstop>AFiyatdoubleSpinBox</tabstop>
  <tabstop>SFiyatdoubleSpinBox</tabstop>
  <tabstop>KDVspinBox</tabstop>
  <tabstop>KDVDahilcomboBox</tabstop>
  <tabstop>OTVspinBox</tabstop>
  <tabstop>OTVDahilcomboBox</tabstop>
  <tabstop>TedarikcicomboBox</tabstop>
  <tabstop>KaydetpushButton</tabstop>
  <tabstop>iptalpushButton</tabstop>
  <tabstop>temizlepushButton</tabstop>
  <tabstop>BarkodOlusturpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
